src/main/modules/SnippetsManager.js

```diff
@@ -195,7 +196,13 @@ class SnippetsManager {
         try {
             return await this._evaluate(matchedString, code)
         } catch (error) {
-            return `QWError: ${_.get('error', 'message', String(error))}`
+            if (!Notification.isSupported()) {
+                return `QWError ${_.get('error', 'message', String(error))}`
+            }
+
+            Notification.show('QWError', _.get('error', 'message', String(error)))
+
+            return ''
```

Comment:
Czy chcemy podmienić na pusty string czy na klucz ?
Jakie zachowanie byłoby bardziej intuicyjne?

Reply:
I think substitution for a trigger may be misleading in a way, people would like to re-trigger the shortcut. Imagine you type `;abcd`, a notification pops up and `;abcd` stays. People may try to remove the last letter and retype it, but it'd have no effect. What do you think?